src/main/java/org/verdictdb/core/querying/simplifier/ConsolidatedExecutionNode.java

```diff
@@ -201,6 +201,19 @@ public SqlConvertible createQuery(List<ExecutionInfoToken> tokens) throws Verdic
     // contains the consolidated select query.
     parentNode.setSelectQuery(selectQuery);
     SqlConvertible consolidatedQuery = parentNode.createQuery(newTokens);
+
+    // Distinguish aggregated column
+    if (parentNode instanceof SelectAllExecutionNode) {
```

Comment:
Would it be possible to have this logic inside QueryNodeBase? At the moment, this logic seems to only work when a plan is simplified.

Reply:
It might be a bit complicated if there are projection nodes on top of AsyncAggExecutionNode. Then it requires ProjectionNode to know some columns are aggregated.

Reply:
I see. I think we should keep some metadata in every node and propagate the information. However, since it requires much additional effort, let's register this as a separate issue without changing this part at the moment.

When you register a new issue, please include a reference to this pull request or discussion.